api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -416,6 +422,33 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
             dbWorkspace.getWorkspaceNamespace(), dbWorkspace.getWorkspaceId(), dbCohort.getCohortId()));
       }
     }
+
+    List<JSONObject> demoConceptSets = cloudStorageService.readAllDemoConceptSets();
+    for (JSONObject conceptSet: demoConceptSets) {
+      ConceptSet dbConceptSet = new ConceptSet();
+      JSONArray conceptIdsJSON = conceptSet.getJSONArray("concept_ids");
+      Set<Long> conceptIds = new HashSet<>();
+      for (int i = 0; i < conceptIdsJSON.length(); i++) {
```

Comment:
Whenever I see a for loop, I want to convert it to a streaming lambda. It generally reads a lot cleaner and saves intermediary collection objects. Something like (untested) `conceptIdsJSON.toStream().map(id -> Long.valueOf(id)).collect(Collectors.toSet());`

Reply:
it seems like (from what i'm reading) this is the most concise way to cast a jsonarray to a set, since there is no way to do the lambda function without error on something that could be any type

Reply:
This compiles for me:
```
      List<Object> conceptIdsList = conceptIdsJSON.toList();
      Set<Long> conceptIds = conceptIdsList
          .stream()
          .map(Object::toString)
          .map(Long::valueOf)
          .collect(Collectors.toSet());
```